beer-garden/react-ui
# Issue/102 Request View

Reworked to bring up to date with conventions and implemented functionality listed on issue #102.
The `pour_it_again` button is broken and will be restored after the command page is finished. Also I renamed the button to `remake request`

Testing

1. make a request that spawns at least 1 child and should be able to navigate to the child request from the parent request and to the parent request from from a child request.

2. Make a request with and without a comment to see the difference between the 2.

3. Make a request with output types of `JSON` and `HTML`, you should be able to toggle between formatted and unformatted output.

4. Try to download the output, should work with any output type
